docs/api.rst

```diff
@@ -1475,6 +1475,7 @@ Query the historical price of assets
 .. http:post:: /api/(version)/assets/prices/historical
 
    Querying this endpoint with a list of lists of asset and timestamp, and a target asset will return an object with the price of the assets at the given timestamp in the target asset currency. Providing an empty list or no target asset is an error.
+   If ``only_cache_period`` is provided, the endpoint will only return prices found in cache within the specified time period around each timestamp. This is useful for scenarios requiring immediate responses, such as graph rendering.
```

Comment:
what is the unit? seconds, minutes, hours?

Reply:
seconds. it's mentioned below in the json body schema definition.
`   :reqjson integer only_cache_period: (Optional) Time period in seconds around each timestamp to search for cached prices. If omitted, falls back to querying historical prices`